python/sglang/srt/managers/scheduler.py

```diff
@@ -868,9 +870,11 @@ def run_batch(self, batch: ScheduleBatch):
         if self.is_generation:
             model_worker_batch = batch.get_model_worker_batch()
             if batch.forward_mode.is_decode() or batch.extend_num_tokens != 0:
-                logits_output, next_token_ids = self.tp_worker.forward_batch_generation(
-                    model_worker_batch
-                )
+                (
+                    logits_output,
+                    next_token_ids,
+                    next_token_embeds,
```

Comment:
Why do you need `next_token_embeds`? I think after the first prefill, we can use token ids and do not need to take embedding inputs anymore.